autopush/tests/test_endpoint.py

```diff
@@ -173,7 +170,7 @@ def test_update_too_much_data(self):
 
         def handle_finish(result):
             self.assertTrue(result)
-            self.status_mock.assert_called_with(401)
+            self.status_mock.assert_called_with(413)
```

Comment:
should we have the tests check for the errnos as well if we're publishing them and clients are relying on them?